tslint.json

```diff
@@ -7,6 +7,8 @@
     "one-line": false,
     "ordered-imports": false,
     "trailing-comma": false,
-    "interface-name" : [false]
+    "interface-name" : [false],
+    "variable-name": [true, "ban-keywords", "check-format", "allow-pascal-case", "allow-leading-underscore"],
+    "no-debugger": false
```

Comment:
Adding rules to allow for `_private` variables, and also `debugger` statements. I'm not sure if there's a way to allow me to develop with `debugger`/`console.log` statements without a rule like this?

Reply:
In my experience, use of debugger is generally uncommon and temporary while debugging a particular issue, and adding `// tslint:disable-line` is reasonable in that case. console.log is another matter. That should either be allowed generally or we should figure out a way to allow it in development but disable it in production.

Reply:
I use `debugger` for mocha tests pretty frequently...I could see just disabling the line when I do, though